### ISSUE TITLE
NVDA will not read conversations in Skype 8.89 split view after moving away from window

### ISSUE THREAD

**Issue by enessaribas** [nvaccess/nvda#14253]
### Steps to reproduce:
1. enable Skype split view
2.  open a conversation window, and press windows m to move to desktop
3.  move back to the conversation window with alt tab
### Actual behavior:
NVDA reads the window title, however will not read any controls or information in window.  This is an NVDA issue, this problem does not occur with narrator or other screen readers

### Expected behavior:
NVDA reads the window normally and allows navigating between controls

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
alpha 26801,42b667b9
#### Windows version:
windows 11 22h2, but also occurs on Windows 10
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by Adriani90** [comment]
@enessaribas I cannot reproduce with NVDA 2023.1 RC1 and Skype 94. Could you please update the versions and report how it is now? Are you still having this issue?

**Comment by Adriani90** [comment]
Please make sure that the split view message window is maximized.

**Comment by enessaribas** [comment]
Microsoft has resolved this in a newer version of Skype. This can be closed now.
